src/controller/CHIPDeviceController.h

```diff
@@ -317,8 +318,7 @@ class DLL_EXPORT DeviceController : public Messaging::ExchangeDelegate,
     //////////// ResolverDelegate Implementation ///////////////
     void OnNodeIdResolved(const chip::Mdns::ResolvedNodeData & nodeData) override;
     void OnNodeIdResolutionFailed(const chip::PeerId & peerId, CHIP_ERROR error) override;
-    void OnCommissionableNodeFound(const chip::Mdns::CommissionableNodeData & nodeData) override;
-    void OnCommissionerFound(const chip::Mdns::CommissionableNodeData & nodeData) override;
+    Mdns::DiscoveredNodeData * GetDiscoveredNodes() override { return mCommissionableNodes; }
```

Comment:
How do we trigger a get commissioners operation versus a get commissionable nodes operation?

Reply:
Get commissioners (or starting commissioner discovery for that matter) can be done by instantiating a CHIPCommissionableNodeController. 
On the other hand, Get Commissionable nodes (or starting Commissionable Node discovery) can be done by instantiating a CHIPDeviceController/Commissioner.